resources/windows/wix/customactions.wxs.erb

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
```

Comment:
this looks like it isn't an erb template... if it's static, we shouldn't indicate it's a template